db-assessment/import_db_assessment.py

```diff
@@ -0,0 +1,833 @@
+            if fileTableCounter == 1:
+
+                # If already exists delete the file
+                if os.path.exists(targetFileNameConsolidated):
+                    
+                    print('The file {} already exists. It is going to be overwritten.'.format(targetFileNameConsolidated))
+                    os.remove(targetFileNameConsolidated)
+
+            # This is the file that will be used to be consolidated
+            fileConsolidated = open(targetFileNameConsolidated,'a')
+
+            # This file was found in the OS. The content of this file will be merged/consolidated into fileConsolidated
+            fileToBeConsolidated = open(fileName, 'r')
+
+            # Breaking it down into lines because first two lines must be skipped for all of the files (expect first file merged)
+            # Since those files are expected to be small (< 10k lines) no performance issue is expected
+            linesToBeConsolidated = []
+            linesToBeConsolidated = fileToBeConsolidated.readlines()
+
+            # To control how many lines are being processed and identify the first processed lines since it needs to skip it eventually
+            lineCounter = 0
+            for line in linesToBeConsolidated:
+
+                # Line counters to be used to skip unecessary lines
+                lineCounter = lineCounter + 1
+
+                # Not processing first lines due to expected CSV headers. Except for the first file.
+                if lineCounter <= 2 and fileTableCounter > 1:
+                    
+                    continue
+                
+                # Writting up the line from linesToBeConsolidated into fileConsolidated
+                fileConsolidated.write(line)
+
+
+            # Closing file handle
+            fileToBeConsolidated.close()
+
+            # Closing file handle
+            fileConsolidated.close()
+
+    print ('\nThe total files consolidated are {}. \nAll files are located in {}'.format(str(fileCounter),str(getattr(args,'fileslocation'))))
+
+    return True
+
+def createOptimusPrimeViews(gcpProjectName,bqDataset):
+# This function intents to create all views found in the opViews directory. The views creation must follow opViews/<filename> order
+
+    print ('\nPreparing to create Optimus Prime SQL Views\n')
+    
+    # Store all files found in the OS
+    fileList = []
+
+    # Searching for all matching files in the default views location
+    filePattern = 'opViews/optimus_createView*.sql'
+
+    # List with all views to be created
+    fileList = getAllFilesByPattern(filePattern)
+    
+    if len(fileList) == 0:
+        print('\nWARNING: No views found to be created at expected location: {}. Please make sure you the location is correct.'.format(filePattern))
+        # Returns False if cannot create views    
+        return False
+    
+    else:
+
+        client = bigquery.Client(client_info=set_client_info.get_http_client_info())
```

Comment:
Accepted. To be included in the next minor release.

Reply:
Since get_bigqueryClient method is already defined this change will be  
``` client = get_bigqueryClient()```

Any reason why to wait until next release?

Reply:
Anything that would require a full test set I am pushing it to the other minor release that most likely will be out there in 1-2 week timeframe. 
For that version, I will have to go a full detailed testing again. So, we just want to optimize this time.